packages/react-components/react-nav-preview/src/components/NavDrawerBody/NavDrawerBody.types.ts

```diff
@@ -1,17 +1,13 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { DrawerBodyProps, DrawerBodySlots, DrawerBodyState } from '@fluentui/react-drawer';
```

Comment:
This messes with treesharking, should be

```suggestion
import type { DrawerBodyProps, DrawerBodySlots, DrawerBodyState } from '@fluentui/react-drawer';
```